src/VisualStudio/Core/Def/Implementation/ProjectSystem/MetadataReferences/VisualStudioMetadataReference.Snapshot.cs

```diff
@@ -82,15 +82,7 @@ private bool SaveMetadataReadingException(Exception e)
 
             protected override DocumentationProvider CreateDocumentationProvider()
             {
-                string xmlDocumentPath;
-                if (ReferencePathUtilities.TryFindXmlDocumentationFile(FilePath, out xmlDocumentPath))
```

Comment:
Funnily enough it looks like we getting lucky in the past passing the XML file to this method anyway - looking at it's implementation, it's expecting a module file path.